src/views/home/UserInput.js

```diff
@@ -39,55 +47,81 @@ export default class UserInput extends Component {
   onFormSubmit = async e => {
     e.preventDefault();
     const body = JSON.stringify(this.state);
-
-    try {
-      const res = await createTicket(body);
-
+    const res = await createTicket(body);
+    if (!res) {
+      console.error('Error creating ticket!!');
+      // TODO: let user know about error
+    } else {
+      console.log(res);
       this.setState({ newTicket: res.ticket });
-    } catch (error) {
-      // TODO: let user know about the error
     }
   };
+
   render() {
     const { newTicket } = this.state;
+    let screenshot = this.state.screenshot;
     return (
       <div>
         {newTicket ? (
-          <Redirect to={`/tickets/${newTicket.ID}`} />
+          <Redirect
+            to={`/tickets/${newTicket.ID}/${screenshot.height}/${screenshot.width}`}
+          />
         ) : (
           <>
             <Header as="h3"> Enter Parameters:</Header>
             <Form
               onSubmit={this.onFormSubmit}
               style={{ display: 'inline-block' }}
             >
-              <Input
-                style={inputStyle}
-                type="url"
-                name="url"
-                required
-                placeholder="http://mysite.com"
-                value={this.state.url}
-                onChange={this.onChange}
-              />
+              <Form.Group inline>
+                <label style={{ padding: '10px' }}>Url:</label>
+                <Form.Input
+                  style={inputStyle}
+                  type="url"
+                  name="url"
+                  required
+                  placeholder="http://mysite.com"
+                  value={this.state.url}
+                  onChange={this.onChange}
+                />
+                <Button content="Submit" />
+                <Button
+                  type="button"
+                  onClick={this.showOptions}
+                  color="teal"
+                  icon
+                >
+                  <Icon name="cog" />
+                  Options
+                </Button>
+              </Form.Group>
 
-              {/*<input
-            style={inputStyle}
-            type="text"
-            name="width"
-            placeholder="Enter an image width"
-            value={this.state.screenshot.width}
-            onChange={this.onChange}
-          />
-          <input
-            style={inputStyle}
-            type="text"
-            name="height"
-            placeholder=" Enter an image height"
-            value={this.state.screenshot.height}
-            onChange={this.onChange}
-          /> */}
-              <Button content="Submit" />
+              {this.state.showOptions && (
+                <Form.Group inline>
+                  <label>Width:</label>
```

Comment:
This is definitely a lower priority, but multiple screenshots / ability to preselect from common sets of user agents + screen sizes would be cool. Maybe something to put on the task board at today's meeting.